Python tests are failing

### How to Reproduce
1. Add a new group 'it'
2. Import it_emx_datatypes
3. Copy the Python REST test from https://github.com/molgenis/molgenis/blob/7.0/molgenis-python/src/test/python/test-rest.py and run it

### Expected behavior
No test failures

### Observed behavior
```
======================================================================
ERROR: test_login_logout_and_get_MolgenisUser (__main__.TestStringMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "rest-test.py", line 26, in test_login_logout_and_get_MolgenisUser
    response = s.get(self.user_entity)
  File "/Users/tommy/temp/molgenis.py", line 110, in get
    return response.json()["items"]
  File "/Library/Python/2.7/site-packages/requests/models.py", line 805, in json
    return complexjson.loads(self.text, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded

======================================================================
FAIL: test_get_attribute_meta (__main__.TestStringMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "rest-test.py", line 123, in test_get_attribute_meta
    meta)
AssertionError: {'labelAttribute': True, 'auto': False, 'visible': True, 'readOnly': False, 'hre [truncated]... != {u'labelAttribute': True, u'isAggregatable': False, u'name': u'username', u'auto [truncated]...
Diff is 926 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_no_login_and_get_MolgenisUser (__main__.TestStringMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "rest-test.py", line 37, in test_no_login_and_get_MolgenisUser
    self.assertEqual(e.response.json()['errors'][0]['message'], self.no_count_permission_user_msg)
AssertionError: u"No 'Read metadata' permission on entity type 'User' with id 'sys_sec_User'." != 'No [COUNT] permission on entity type [User] with id [sys_sec_User]'

----------------------------------------------------------------------
Ran 10 tests in 9.358s

FAILED (failures=2, errors=1)
```

--- a/docs/developer_documentation/ref-python.md
+++ b/docs/developer_documentation/ref-python.md
@@ -10,7 +10,7 @@ Now you can create a python script.
 To get started, you should import the python api, connect to a molgenis server and login:
 ```python
 import molgenis
-session = molgenis.Session.("https://molgenis.mydomain.example/api/")
+session = molgenis.Session("https://molgenis.mydomain.example/api/")
 session.login("username","password")
 ```
 Always put the import and molgenis.Session in your script to make the api work. 
@@ -18,7 +18,7 @@ Always put the import and molgenis.Session in your script to make the api work.
 # Overview example
 ```python
 import molgenis
-session = molgenis.Session.("https://molgenis.mydomain.example/api/")
+session = molgenis.Session("https://molgenis.mydomain.example/api/")
 session.login("username","password")
 my_table = session.get("package_entityName")
 print(my_table)

--- a/molgenis-python/src/main/resources/templates/molgenis.py.ftl
+++ b/molgenis-python/src/main/resources/templates/molgenis.py.ftl
@@ -45,6 +45,7 @@ class Session():
                                      headers={"Content-Type": "application/json"})
         if response.status_code == 200:
             self.token = response.json()["token"]
+
         response.raise_for_status()
         return response
 
@@ -54,6 +55,7 @@ class Session():
                                      headers=self._get_token_header())
         if response.status_code == 200:
             self.token = None
+            self.session.cookies.clear()
         response.raise_for_status()
         return response
 

--- a/molgenis-python/src/main/resources/templates/molgenis.py.ftl
+++ b/molgenis-python/src/main/resources/templates/molgenis.py.ftl
@@ -199,7 +199,7 @@ class Session():
         url = self.url.strip('/api/') + '/plugin/importwizard/importFile'
         response = requests.post(url, headers=header, files=files)
         if response.status_code == 201:
-            return response.json()
+            return response.content
         response.raise_for_status()
         return response
 

--- a/molgenis-python/src/test/python/test-rest.py
+++ b/molgenis-python/src/test/python/test-rest.py
@@ -7,7 +7,7 @@ import molgenis
 class TestStringMethods(unittest.TestCase):
     api_url = "https://molgenis62.gcc.rug.nl/api/"
 
-    no_count_permission_user_msg = 'No [COUNT] permission on entity type [User] with id [sys_sec_User]'
+    no_readmeta_permission_user_msg = "No 'Read metadata' permission on entity type 'User' with id 'sys_sec_User'."
     user_entity = 'sys_sec_User'
     ref_entity = 'it_emx_datatypes_TypeTestRef'
 
@@ -20,13 +20,13 @@ class TestStringMethods(unittest.TestCase):
     def test_login_logout_and_get_MolgenisUser(self):
         s = molgenis.Session(self.api_url)
         s.login('admin', 'admin')
-        response = s.get(self.user_entity)
+        s.get(self.user_entity)
         s.logout()
         try:
-            response = s.get(self.user_entity)
+            s.get(self.user_entity)
         except requests.exceptions.HTTPError as e:
             self.assertEqual(e.response.status_code, 401)
-            self.assertEqual(e.response.json()['errors'][0]['message'], self.no_count_permission_user_msg)
+            self.assertEqual(e.response.json()['errors'][0]['message'], self.no_readmeta_permission_user_msg)
 
     def test_no_login_and_get_MolgenisUser(self):
         s = molgenis.Session(self.api_url)
@@ -34,7 +34,7 @@ class TestStringMethods(unittest.TestCase):
             s.get(self.user_entity)
         except requests.exceptions.HTTPError as e:
             self.assertEqual(e.response.status_code, 401)
-            self.assertEqual(e.response.json()['errors'][0]['message'], self.no_count_permission_user_msg)
+            self.assertEqual(e.response.json()['errors'][0]['message'], self.no_readmeta_permission_user_msg)
 
     def test_add_all(self):
         s = molgenis.Session(self.api_url)
@@ -115,12 +115,13 @@ class TestStringMethods(unittest.TestCase):
         s = molgenis.Session(self.api_url)
         s.login('admin', 'admin')
         meta = s.get_attribute_meta_data(self.user_entity, 'username')
-        self.assertEqual({'isAggregatable': False, 'attributes': [], 'auto': False,
-                          'fieldType': 'STRING', 'href': '/api/v1/' + self.user_entity + '/meta/username',
-                          'label': 'Username',
-                          'labelAttribute': True, 'lookupAttribute': True, 'maxLength': 255, 'nillable': False,
-                          'readOnly': False, 'unique': True, 'visible': True, 'name': 'username', 'enumOptions': []},
+        self.assertEqual({'labelAttribute': True, 'isAggregatable': False, 'name': 'username',
+                          'auto': False, 'nillable': False, 'label': 'Username', 'lookupAttribute': True,
+                          'visible': True, 'readOnly': True, 'href': '/api/v1/sys_sec_User/meta/username',
+                          'enumOptions': [], 'fieldType': 'STRING', 'maxLength': 255, 'attributes': [],
+                          'unique': True},
                          meta)
+
 
 if __name__ == '__main__':
     unittest.main()

